### PR TITLE
skip block sdk, params migration

### PR DESCRIPTION
**Pull request by cgsingh33**
- skip-mev's Block sdk 
- go version bump to 1.21+
- testnet v14 upgrade handler
- wasmd, wasmvm bump
- TxSizeCostPerByte 2x
- code cleanup

### CODE REVIEW

File: app/wasm/testdata/counter.wasm
Comment (dheerajkd30) [inline app/wasm/testdata/counter.wasm:1]: shouldn't this be excluded  ?

Comment (cgsingh33) [reply app/wasm/testdata/counter.wasm:1]: this is for wasm test case

File: x/esm/types/codec.go [hunk 1]
```diff
@@ -39,6 +44,7 @@ var (
 func init() {
 	RegisterCodec(Amino)
```
Comment (dheerajkd30) [inline x/esm/types/codec.go]: Why two Registercodec ?

Comment (dheerajkd30) [reply x/esm/types/codec.go]: please check at other codex files too.

File: x/liquidationsV2/types/codec.go [hunk 2]
```diff
@@ -39,6 +44,7 @@ var (
 func init() {
 	RegisterLegacyAminoCodec(amino)
```
Comment (dheerajkd30) [inline x/liquidationsV2/types/codec.go:42]: RegisterCodec or RegisterLegacyAminoCodec ? This should be common across module

**Review by dheerajkd30 — CHANGES_REQUESTED** [review]
Go through the comments please.

File: x/esm/types/codec.go (hunk 1 above)
Comment (cgsingh33) [reply x/esm/types/codec.go]: fixed

File: x/liquidationsV2/types/codec.go (hunk 2 above)
Comment (cgsingh33) [reply x/liquidationsV2/types/codec.go:42]: done

**Review by dheerajkd30 — APPROVED** [review]

**Review by pratikasr — APPROVED** [review]

**Review by chaitanyaprabhu10 — APPROVED** [review]

**Pull request merged**